central/integrationhealth/datastore/datastore_impl.go

```diff
@@ -21,91 +21,81 @@ type datastoreImpl struct {
 }
 
 func (ds *datastoreImpl) GetRegistriesAndScanners(ctx context.Context) ([]*storage.IntegrationHealth, error) {
-	if ok, err := integrationSAC.ReadAllowed(ctx); err != nil {
```

Comment:
By doing this we(inadvertently?) make these endpoints return errors to FE. In case of no permissions, that means that now UI will show an error instead of showing an empty health widget. I wonder if that's what we want

Reply:
Hm, yeah. I guess I don't see a reason why it shouldn't have been an error beforehand, but it might break UI in this case.
I reverted the `sac.VerifyAuthZOK` changes.

(unrelated: I feel we should just unify the datastores behavior and be clear: no permission == sac err returned)